packages/@react-spectrum/utils/src/Slots.tsx

```diff
@@ -34,10 +34,11 @@ export function cssModuleToSlots(cssModule) {
   }, {});
 }
 
+const emptyObj = {};
```

Comment:
just to prevent anything from accidentally ending up inside this object and affecting everything, I think the component should create this object

```
let emptyObj = useMemo(() => ({}), []);
```

then at least it's somewhat confined

for instance, this could happen https://codesandbox.io/p/sandbox/tkdd2j